portfolio/src/main/webapp/script.js

```diff
@@ -50,7 +50,7 @@ const GALLERY_ITEMS = [
   }
 ];
 
-const TIME_INTERVAL = 10000; // Constant time of 10 seconds
+const TIME_INTERVAL = 5000; // Constant time of 10 seconds
```

Comment:
Update the comment. :)
```suggestion
const TIME_INTERVAL = 5000; // Constant time of 5 seconds
```

Reply:
Thanks for pointing out!